### DISCUSSION TITLE
Downstream encryption?

### DISCUSSION THREAD

**Discussion by npdgm** [Anime4000/RTL960x#157] [General]
Hi,
I'm using DFP-34X-2C2.

Has anyone figured out how to check whether encryption of the downstream is active or not? I've seen this status on Lantiq based sticks before, but can't find it with RTL tools.

And did you ever get encryption to work? I've no idea what's required for the OLT to negotiate it. That must be very ISP specific, but any information would be much appreciated, thanks.

Cheers

**Comment by Anime4000** [comment]
This one I not sure how to check, at least `diag` show this
```
RTK.0> gpon show us-flow
============================================================
   GPON ONU MAC U/S Flow Status
Flow ID | GEM Port | Type | TCont
     0 |      263 |  ETH |     0
     1 |      264 |  ETH |     1
     2 |      267 |  ETH |     1
     3 |      272 |  ETH |     2
    64 |        2 | OMCI |    16
===========================================================
RTK.0> port get status port all
Port Status Speed    Duplex TX_FC RX_FC
---- ------ -----    ------ ----- -----
0    Up     1000M    Full   En    En
2    Up     1000M    Full   Dis   Dis
RTK.0> gpon get aes-framecnt

 cparser_cmd_gpon_get_aes_framecnt 2573
```
